### PR TITLE
Replacing IMDb with TMDb

### PR DESCRIPTION
**Pull request by Marc-Andrieu**
IMDb > TMDb, unless it's related to pasting the IMDb link on the admin page

### CODE REVIEW

**Review by julien4215 — APPROVED** [review]

File: app/modules/cinema/endpoints_cinema.py [hunk 1]
```diff
@@ -82,19 +82,19 @@ async def get_movie(
...
+                    f"Code {response.status_code} for TMDb request with movie ID {themoviedb_id}. JSON response: {response.json()}",
                 )
                 raise HTTPException(
                     status_code=500,
                     detail=f"Unknown error (code {response.status_code})",
                 )
     except httpx.RequestError as error:
         hyperion_error_logger.error(error)
-        raise HTTPException(status_code=504, detail="Could not reach the IMdB server")
+        raise HTTPException(status_code=504, detail="Could not reach the TMdB server")
```
Comment (armanddidierjean) [inline app/modules/cinema/endpoints_cinema.py:97]:
```suggestion
        raise HTTPException(status_code=504, detail="Could not reach the TMDB server")
```

File: app/modules/cinema/endpoints_cinema.py [hunk 2]
```diff
@@ -82,19 +82,19 @@ async def get_movie(
             case 404:
                 raise HTTPException(
                     status_code=404,
-                    detail=f"Movie not found for IMDb movie ID {themoviedb_id} (code {response.status_code})",
+                    detail=f"Movie not found for TMDb movie ID {themoviedb_id} (code {response.status_code})",
                 )
             case _:
                 hyperion_error_logger.error(
-                    f"Code {response.status_code} for IMDb request with movie ID {themoviedb_id}. JSON response: {response.json()}",
+                    f"Code {response.status_code} for TMDb request with movie ID {themoviedb_id}. JSON response: {response.json()}",
```
Comment (armanddidierjean) [inline app/modules/cinema/endpoints_cinema.py:89]:
```suggestion
                    f"Code {response.status_code} for TMDB request with movie ID {themoviedb_id}. JSON response: {response.json()}",
```

File: app/modules/cinema/endpoints_cinema.py [hunk 3]
```diff
@@ -82,19 +82,19 @@ async def get_movie(
             case 404:
                 raise HTTPException(
                     status_code=404,
-                    detail=f"Movie not found for IMDb movie ID {themoviedb_id} (code {response.status_code})",
+                    detail=f"Movie not found for TMDb movie ID {themoviedb_id} (code {response.status_code})",
```
Comment (armanddidierjean) [inline app/modules/cinema/endpoints_cinema.py:85]:
```suggestion
                    detail=f"Movie not found for TMDB movie ID {themoviedb_id} (code {response.status_code})",
```

File: app/modules/cinema/endpoints_cinema.py [hunk 4]
```diff
@@ -73,7 +73,7 @@ async def get_movie(
                 return schemas_cinema.TheMovieDB(**response.json())
             case 401:
                 hyperion_error_logger.error(
-                    f"INVALID API KEY - Code 401 for IMDb request. JSON  response: {response.json()}",
+                    f"INVALID API KEY - Code 401 for TMDb request. JSON  response: {response.json()}",
```
Comment (armanddidierjean) [inline app/modules/cinema/endpoints_cinema.py:76]:
```suggestion
                    f"INVALID API KEY - Code 401 for TMDB request. JSON  response: {response.json()}",
```

**Comment by Marc-Andrieu** [comment]
@armanddidierjean

**Review by armanddidierjean — APPROVED** [review]

**Pull request merged**